docs/feature/storage-backends/aws-secrets-manager.md

```diff
@@ -0,0 +1,165 @@
+            inject:
+              requiredTypeInstances:
+              - id: ${TI_ID}
+                description: "AWS credentials"
+       - interface: 
+           path: cap.*
+         oneOf:
+         - implementationConstraints:
+             requires:
+             - path: cap.core.type.platform.kubernetes
+         - implementationConstraints: {}
+   typeInstance:
+     rules: [] 
+   ENDOFFILE
+   capact policy apply -f /tmp/aws-storage-policy.yaml
+   ```
+
+1. Create parameters:
+
+   ```bash
+   cat > /tmp/aws-storage-input-params.yaml << ENDOFFILE
+   input-parameters:
+    region: "eu-west-1" # AWS region, where the secrets are managed
+   ENDOFFILE
+   ```
+
+1. Create Action:
+   ```bash
+   capact act create cap.interface.aws.secrets-manager.storage.install --name aws-storage -n capact-system --parameters-from-file /tmp/aws-storage-input-params.yaml
+   ```
+
+1. Wait for the Action to have the `READY_TO_RUN` status:
+
+   ```bash
+   capact act get aws-storage -n capact-system
+   ```
+
+1. Run Action:
+
+   ```bash
+   capact act run aws-storage -n capact-system
+   ```
+
+1. Watch the Action for completion:
+
+    ```bash
+    capact act watch aws-storage -n capact-system
+    ```
+
+1. Get output TypeInstances:
+
+    ```bash
+    capact act get aws-storage -n capact-system -ojson | jq '.Actions[0].output.typeInstances'
+    ```
+
+    See the details of the installed AWS storage backend:
+
+     ```bash
+     AWS_SM_STORAGE_ID=$(capact act get aws-storage -n capact-system -ojson | jq '.Actions[0].output.typeInstances | map(select(.typeRef.path == "cap.type.aws.secrets-manager.storage"))[0].id' -r)
+     capact ti get $AWS_SM_STORAGE_ID -oyaml
+     ```
+
+## Use the storage backend
+
+Once the storage backend is installed, you can configure it for selected set of TypeInstances.
+
+For example, you can configure it as a default backend for all TypeInstances with the following Global Policy:
+
+```bash
+cat > /tmp/aws-storage-policy.yaml << ENDOFFILE
+interface:
+  rules:
+      - interface:
+          path: cap.*
+        oneOf:
+        - implementationConstraints:
+            requires:
+            - path: cap.core.type.platform.kubernetes
+        - implementationConstraints: {}
+typeInstance:
+   rules:
+     - typeRef:
+         path: "cap.*"
+       backend:
+         id: "${AWS_SM_STORAGE_ID}"
+         description: "AWS Secrets Manager"
+ENDOFFILE
+capact policy apply -f /tmp/aws-storage-policy.yaml
+```
+
+To read more about TypeInstance backend configuration, see the [Definition of rules for TypeInstance](../policies/overview.md#definition-of-rules-for-typeinstance) section.
+
+## Next steps
+
+Now, you can run any Capact manifest and utilize the newly installed AWS Secrets Manager storage backend. For example, [install Mattermost](../../example/mattermost-installation.md). Once it's installed, verify the output TypeInstance values are created in AWS Secrets Manager. You can do it in two ways:
```

Comment:
```suggestion
Now, you can run any Capact manifest and utilize the newly installed AWS Secrets Manager storage backend. For example, [install Mattermost](../../example/mattermost-installation.md). Once it's installed, verify the output TypeInstance values are created in AWS Secrets Manager. 
You can do it in two ways:
```

Reply:
I don't want to do the line break as is the logical continuation of previous sentence (you can do **it**). But I tried to tweak it a little 🙂